source/opt/ir_context.h

```diff
@@ -24,7 +25,18 @@ namespace ir {
 
 class IRContext {
  public:
-  IRContext(std::unique_ptr<Module>&& m) : module_(std::move(m)) {}
+  enum Analysis {
+    kAnalysisNone = 0x0,
+    kAnalysisFirst = 0x1,
```

Comment:
Quibbles:
 
Seems kAnalysisFirst should be kAnalysisBegin = 0x1, and then kAnalysisDefUse = kAnalysisBegin

Also  kAnalysisLast should be renamed kAnalysisEnd  ?

Reply:
done